react/__tests__/components/CategoriesHighlights.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import { render } from 'enzyme'
+
+import CategoriesHighlights from '../../CategoriesHighlights'
+
+describe('<CategoriesHighlights />', () => {
+  const defaultProps = {
```

Comment:
There is no need to defaultProps here, it retrieve the default props of the component